src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -123,6 +123,13 @@ std::vector<ActorID> GcsActorScheduler::CancelOnNode(const ClientID &node_id) {
   return actor_ids;
 }
 
+void GcsActorScheduler::CancelLeasingRequest(const ClientID &node_id,
```

Comment:
The function name CancelLeasingRequest is confusing because it doesn't actually cancel the request, it just ignores the result that the request returned.

Reply:
Will changing to `CancelOnLeasing` following other conventions.